prettyprinter/src/Data/Text/Prettyprint/Doc/Internal.hs

```diff
@@ -1785,7 +1794,16 @@ layoutWadlerLeijen
       | fits pWidth minNestingLevel availableWidth x = x
       | otherwise = y
       where
-        minNestingLevel = min lineIndent currentColumn
```

Comment:
Any idea why this was computed with `min` and didn't just return `lineIndent`?

Doesn't the invariant from above hold here?

    -- * current column >= current nesting level